workers/unity/Assets/Gdk/Core/Components/WorldCommandsTranslation.cs

```diff
@@ -352,7 +352,7 @@ private bool TryGetEntityFromRequestId(uint requestId, string responseName, out
             long entityId;
             if (!RequestIdToEntityId.TryGetValue(requestId, out entityId))
             {
-                Debug.LogErrorFormat(TranslationErrors.CannotFindEntityForCommandResponse, requestId, responseName);
+                Debug.LogErrorFormat(TranslationErrors.RequestDoesNotExist, requestId, responseName);
```

Comment:
we need to make sure that this goes in before Martin's PR OR rebase on top of it, if his goes in first to ensure that we don't start using `LogError` again

Reply:
I suspect this one will be ready first. In which case the only change to the other ones will be a string change.